docs/BUILD_AND_TEST.md

```diff
@@ -1,132 +1,105 @@
- 
-# Build and Test
-## Getting Started
+<!--
+http://www.apache.org/licenses/LICENSE-2.0.txt
 
-To build snap you'll need:
-
-* [Golang >= 1.4](https://golang.org)
-    * Golang should be [downloaded](https://golang.org/dl/), and [installed](https://golang.org/doc/install)
-      The below instructions assume that the GOPATH environment variable has been set.
-    * An option to look into is using the [go version manager (gvm)](https://github.com/moovweb/gvm) if you want to easily switch between Go versions.
-* [GNU Make](https://www.gnu.org/software/make/)
-* [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-* snap:
-    * Install snap into your `$GOPATH`.
 
-    ```
-    go get github.com/intelsdi-x/snap`.
-    ```
+Copyright 2015 Intel Corporation
 
-    * `cd` into `$GOPATH/src/github.com/intelsdi-x/snap` and set your fork as a remote repository.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
 
-    ```
-    git remote rename origin upstream
-    ```
+    http://www.apache.org/licenses/LICENSE-2.0
 
-    ```
-    git remote add origin git@github.com:<yourGithubID>/snap.git
-    ```
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+--> 
+# Build and Test
+This guide gets you started with building and testing Snap. If you have commits you want to contribute, review the [CONTRIBUTING file](../CONTRIBUTING.md) for a shorter list of what we look for and come back here if you need to verify your environment is configured correctly.
 
-    * To push to your fork later, either do a `git push origin master` or do `git checkout -b <someBranchName>` then you can do `git push`.
+## Getting Started
+If you prefer a video walkthrough this process, watch this video: https://vimeo.com/161561815
 
-## Build
+To build the Snap Framework you'll need:
+* [Golang >= 1.5](https://golang.org)
+    * Should be [downloaded](https://golang.org/dl/) and [installed](https://golang.org/doc/install)
```

Comment:
Follow the download and installation instructions on the Golang website sounds a little better